src/config-schema.js

```diff
@@ -55,6 +55,25 @@ const configSchema = {
           }
         }
       },
+      uriHandlerRegistration: {
```

Comment:
More naming feedback... I'm seeing a mix of "URL" and "URI". I prefer URI myself because I think it's a more general concept.